SWLOR.Game.Server/Feature/StatusEffectDefinition/PremonitionStatusEffect.cs

```diff
@@ -45,11 +45,11 @@ private void Premonition1()
                 .CannotReplace(StatusEffectType.Premonition2)
                 .GrantAction((source, target, length, effectData) =>
                 {
-                    Impact(source, 5);
+                    Impact(source, 15);
```

Comment:
You need to update the Perk definition to reflect this change. Also need updates to the Bible.

Reply:
Fixed the perk definition, but not able to touch the bible. I know Skill issue.